easybuild/easyconfigs/o/Octave/Octave-7.1.0-foss-2021b.eb

```diff
@@ -0,0 +1,78 @@
+name = 'Octave'
+version = '7.1.0'
+
+homepage = 'https://www.gnu.org/software/octave/'
+description = """GNU Octave is a high-level interpreted language, primarily intended for numerical computations."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['d4a9d81f3f67b4a6e07cb7a80dcb10ad5e9176fcc30762c70a81580a64b8b0b6']
+
+builddependencies = [
+    ('Bison', '3.7.6'),
+    ('flex', '2.6.4'),
+    ('Autotools', '20210726'),
+    ('gperf', '3.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('X11', '20210802'),
+    ('PCRE', '8.45'),
+    ('ncurses', '6.2'),
+    ('libreadline', '8.1'),
+    ('arpack-ng', '3.8.0'),
+    ('cURL', '7.78.0'),
+    ('FLTK', '1.3.7'),
+    ('fontconfig', '2.13.94'),
+    ('freetype', '2.11.0'),
+    ('GLPK', '5.0'),
+    ('GL2PS', '1.4.2'),
+    ('gnuplot', '5.4.2'),
+    ('Java', '17', '', True),
+    ('zlib', '1.2.11'),
+    ('Mesa', '21.1.7'),
+    ('libGLU', '9.0.2'),
+    ('Qhull', '2020.2'),
+    ('Qt5', '5.15.2'),
+    ('HDF5', '1.12.1'),
+    ('qrupdate', '1.1.2'),
+    ('SuiteSparse', '5.10.1', '-METIS-5.1.0'),
+    ('texinfo', '6.8'),
```

Comment:
@sassy-crick Does Octave actually need `texinfo` itself, or only `makeinfo`?

`texinfo` is a bit nasty because it requires `texlive` as OS dependency...

Reply:
I have tried and I could remove it. Thanks for pointing that out.